modules/calparse.py

```diff
@@ -3,29 +3,33 @@
 from datetime import datetime
 import re
 
+
 class ParsedCal():
     """A calendar parser object"""
 
     def __init__(self, raw):
         """Initialize parsed calendar's objects"""
         self.raw = raw
+        self.courtdates = []
+        self.time_format = '%I:%M%p'
```

Comment:
I think this will work better, thanks! I do wonder if we'll encounter courts that will have different date formats though. Just have to address that as it comes up.